### DISCUSSION TITLE
spinner can be exploited by simply jittering your mouse

### DISCUSSION THREAD

**Discussion by Faustas156** [ppy/osu#16340] [Q&A] [answered]
the title explains it all, but whenever you jitter your mouse, you can get around 600+ SPM whilst not even spinning properly

here's a video of the issue: https://youtu.be/8JFcj21DXSs

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/8224

**Reply by Faustas156** [reply]
these were mostly done on mobile, but i don't think i've seen one being showcased for PC

**Reply by bdach** [reply]
doesn't matter, same issue
